n2y/notion.py

```diff
@@ -84,6 +116,21 @@ def __init__(
         self.load_plugins(plugins)
         self.plugin_data = {}
 
+    @property
+    def retry_api_calls(self):
+        return self.max_retries > self.retry_count
+    
+    @property
+    def max_retries(self):
```

Comment:
@bimbashrestha and @sHermanGriffiths I was thinking the "max retries" should apply to a single API call. Thus, if an API call gets a 429, it will retry that same API call up to 5 times.

If the next API call after that gets a 429, it could also get up to 5 retries before an exception is raised. Is that the behavior you guys were expecting?

Reply:
Yes, I forgot to reset the count. Just finished doing that

Reply:
Nice catch. Yeah, each api call should definitely be treated independently in terms of retries.